loader/LoaderAndLayerInterface.md

```diff
@@ -2692,12 +2692,12 @@ Additional Notes:
 - The loader will filter out extensions requested in `vkCreateInstance` and
 `vkCreateDevice` before calling into the ICD; Filtering will be of extensions
 advertised by entities (e.g. layers) different from the ICD in question.
-- The loader will not call the ICD for `vkEnumerate\*LayerProperties`() as layer
+- The loader will not call the ICD for `vkEnumerate*LayerProperties`() as layer
```

Comment:
You didn't create the issue, but I would think we should probably remove the parenthesis after the function name

Reply:
Will do.